src/app/examples/list-example/list-example.component.ts

```diff
@@ -9,6 +9,7 @@ export class ListExampleComponent implements OnInit {
 
   myItems = [
     {title: 'Vestas Wind Systems', subTitle: '20 stk', amount: '18.309 DKK', detail: 225},
+    // "detail" is missing to demonstrate how line height adjusts to the content
```

Comment:
Could it be an idea to add the details property again. But maybe with 0 value and then make a check in the view code to not render that line of txt if the value is 0 ? 

That way we leave the data intact.

Reply:
not a bad idea